src/app/shared/components/image-form-control/image-form-control.component.ts

```diff
@@ -118,7 +118,39 @@ export class ImageFormControlComponent implements OnInit, ImageFormControlCompon
     }
   }
 
-  public fileChangeEvent(event: string): void {
+  public fileChangeEvent(event: Event): void {
+    const target = event.target as HTMLInputElement;
+    this.imageDecoder(target.files[0], (ev: ProgressEvent<FileReader>) => {
+      const img = new Image();
+      img.src = ev.target.result as string;
+      img.onload = (): void => {
+        const config = this.cropperConfig;
+        if (img.width < config.cropperMinWidth || img.height < config.cropperMinHeight) {
+          this.store.dispatch(
```

Comment:
I would like to move the part with displaying the error and clearing the input to separate method